File: algoexpert/easy/javascript/Bubble_sort.js
const bubbleSort = nums => {
	for (let i = 0; i < nums.length; i++) {
		for (let j = i + 1; j < nums.length; j++) {
			if (nums[i] >= nums[j]) {
				let tmp = nums[i];
				nums[i] = nums[j];
				nums[j] = tmp;
			}
		}
	}
	return nums;
}

console.log(bubbleSort([2, 3, 1, 5, 4, 8, 7]));

File: algoexpert/easy/python/Bubble_sort.py
def bubble_sort(arr):
	for i in range(len(arr)):
		for j in range(i, len(arr)):
			if arr[i] > arr[j]:
				tmp = arr[i]
				arr[i] = arr[j]
				arr[j] = tmp

	return arr


print(bubble_sort([ 1, 2, 7, 3, 5, 4, 8]))
